### ISSUE TITLE
Support branch coverage measurement in the coverage PR comment

### ISSUE THREAD

**Issue by kevalmorabia97** [MishaKav/pytest-coverage-comment#110]
Feature request: Is it possible to have a config parameter to show the table with branch coverage measurement as well?
Currently the coverage percentage does not consider branch percentage. But if someone uses `--branch` flag in the coverage command or `branch=True` in `setup.cfg`, the table that this action posts would not be consistent with the table that pytest-cov prints out
![image](https://user-images.githubusercontent.com/28916987/214135642-003577c5-8b70-4797-8583-fa02a00545e0.png)

**Comment by MishaKav** [comment]
Good idea, I will check the possibility to add this feature when got some time.

**Comment by MishaKav** [comment]
I checked the ability and even tried to add this feature.
For now, it's tough, because it is written in pure javascript without any tests, so I scary to break things.

I think, that first of all when I will get some time, I will try to rewrite this action in `typescript` with strong UTs, then I will be able to add new powerful features like that without feeling that I may break things.

**Comment by vgpastor** [comment]
+1

**Comment by PeterKogan** [comment]
+1

This action is great and that's the only feature I'm missing

**Comment by MishaKav** [comment]
Branch coverage support has been released in [v1.4.0](https://github.com/MishaKav/pytest-coverage-comment/releases/tag/v1.4.0). The action now auto-detects Branch and BrPart columns from both text (`--cov-branch`) and XML coverage formats — no configuration needed.
